src/Map/src/Twig/MapRuntime.php

```diff
@@ -70,7 +76,7 @@ public function renderMap(
 
     public function render(array $args = []): string
     {
-        $map = array_intersect_key($args, ['map' => 0, 'markers' => 0, 'polygons' => 0, 'center' => 1, 'zoom' => 2]);
+        $map = array_intersect_key($args, ['map' => 0, 'markers' => 0, 'polygons' => 0, 'polylines' => 0, 'center' => 1, 'zoom' => 2]);
```

Comment:
I think there is an issue here, I don't fully remember why it has been done like this, but values should be sequential `0, 1, 2, 3, 4, ...`

Reply:
No need here, as it's then using named arguments with the spread.
 
 But it's a good idea at least to prevent future errors if we change to some array_reverse :)